src/pages/match/MatchCandidatesTable.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import DataDisplay from '../../components/dataDisplays/DataDisplay';
+import { cellRendererTypes } from '../../components/dataDisplays/cellRenderers';
+
+const columns = [
+  {
+    name: 'index',
+    labelId: 'ANNOTATION',
+    options: {
+      customBodyRender: i => `C${i + 1}`,
```

Comment:
Why did we pick C? No complaints. Just curious.

Reply:
C is for candidate!